orbstation/antagonists/traitor/objectives/kidnapping.dm

```diff
@@ -0,0 +1,284 @@
+/datum/traitor_objective_category/kidnapping
+	name = "Kidnap Personnel"
+	objectives = list( //Similar weights to destroy heirloom objectives
+		/datum/traitor_objective/kidnapping/common = 4,
+		/datum/traitor_objective/kidnapping/rare = 2,
+		/datum/traitor_objective/kidnapping/captain = 1
+	)
+
+#define KIDNAP_BASE_PROG_REWARD 15 MINUTES
+
+/datum/traitor_objective/kidnapping
```

Comment:
I was interested in trying this out but didn't get around to it, does this just redefine overwrite the tg version from one in our local files?
I was worried that this would produce duplicate-definition errors, but if this works it will be very convenient for us making downstream changes.